src/Dev/SapphireTest.php

```diff
@@ -1887,7 +1887,7 @@ protected function objFromFixture($className, $identifier)
      * Once loaded, you can use idFromFixture() and objFromFixture() to get items from the fixture.
      * Doesn't clear existing fixtures.
      * @param string $fixtureFile The location of the .yml fixture file, relative to the site base dir
-     * @deprecated 4.0.0:5.0.0
+     * @deprecated 4.0.0
```

Comment:
The version in the below notice needs updating, and the message needs to be standardised
And add the message here.